contrib/babelfishpg_common/src/varbinary.c

```diff
@@ -190,6 +203,12 @@ varbinaryin(PG_FUNCTION_ARGS)
 		PG_RETURN_BYTEA_P(result);
 	}
 
+	if ((!(*pltsql_protocol_plugin_ptr && (*pltsql_protocol_plugin_ptr)->is_tds_client)) && 
+		(len >= 4 && inputText[0] == '\\')) /* COPY command from DMS, use base byteain() directly */
```

Comment:
Can you explain why are you checking for an arbit len >=4?

Reply:
This was a safeguard since any octal data must be at minimum `\xxx', so I wanted to a) avoid a segfault in case the string is null or invalid, and b) ensure that it is being executed against a somewhat sane string

Reply:
Can you create a macro rather than hardcoded 4

Reply:
added a macro for this